syntax = "proto3";

import "messages.proto";
option go_package = "github.com/ryepropgroup/protoServer/protos/service";

service Service {
    // stream of sensor data from engine-computer to Go
    rpc SensorDataStream (stream SensorData) returns (Response) {}

    // stream of commands from Go to engine-computer
    rpc CommandStream (DeviceInformation) returns (stream SequenceCommand) {}
}